src/modules/api.js

```diff
@@ -0,0 +1,14 @@
+// const createGame = () => {
+//   const create = async () => {
+//     const get = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+//       {
+//         method: 'post',
+//         body: JSON.stringify({ name: 'leaderboard game' }),
+//         headers: { 'Content-type': 'application/json' },
+//       });
+//     const res = await get.json();
+//     return res;
+//   };
+// };
+
+// export default createGame;
```

Comment:
- [ ] This file is not being used in your program and hence, it would be nice if you could delete it instead of having unused codes commented on it. It's a good practice and professional conduct of having our codebase clean and not have commented unused codes. Please remove this file as it does nothing in your app. 👍🏽